src/fparser/scripts/parse.py

```diff
@@ -65,6 +65,10 @@
 
 from fparser.scripts.script_options import set_parse_options
 import fparser.common.sourceinfo
+
+import logging
```

Comment:
Again, pylint complains this import is in the wrong place.

Reply:
Again, my mistake. Fixed.